source/setup.py

```diff
@@ -239,32 +259,33 @@ def _genManifestTemplate(shouldHaveUIAccess: bool) -> tuple[int, int, bytes]:
 		(".", glob("../miscDeps/python/*.dll")),
 		(".", ['message.html']),
 		(".", [os.path.join(sys.base_prefix, "python3.dll")]),
-	] + (
-		getLocaleDataFiles()
-		+ getRecursiveDataFiles("synthDrivers", "synthDrivers",
-			excludes=tuple(
-				"*%s" % ext
-				for ext in importlib.machinery.SOURCE_SUFFIXES + importlib.machinery.BYTECODE_SUFFIXES
+		] + (
+	getLocaleDataFiles()
+	+ getRecursiveDataFiles(
+		"synthDrivers",
+		"synthDrivers",
+		excludes=tuple(
+			f"*{ext}" for ext in importlib.machinery.all_suffixes()
 			) + (
-				"*.exp",
-				"*.lib",
-				"*.pdb",
-				"__pycache__"

```

Comment:
was pycache removed intentionally from these?

Reply:
Yes, because `fnmatch.fnmatch` won't match `__pycache__`, it needs a wildcard match. Example:

```python
>>> import fnmatch
>>> fnmatch.fnmatch("synthDrivers/__pycache__", "__pycache__")
False
>>> fnmatch.fnmatch("synthDrivers/__pycache__", "*__pycache__")
True
```